fe/lib/components/Questionnaire/QuestionnaireBuilder/FormBuilder/components/NewQuestionForm/components/SelectOptions.tsx

```diff
@@ -65,26 +64,23 @@ export default ({ options, setOptionList }: SelectOptionsProps) => {
 
   return (
     <Stack space="medium">
-      <form onSubmit={submitHandler}>
```

Comment:
This `form` wasn't used (notice that the `Button` below duplicates the listener rather than specifying `type=submit`) and should be avoided since this is nested inside another form.